lib/rules/jsx-props-no-multi-spaces.js

```diff
@@ -27,12 +27,16 @@ module.exports = {
     const sourceCode = context.getSourceCode();
 
     function getPropName(propNode) {
-      if (propNode.type === 'JSXSpreadAttribute') {
-        return sourceCode.getText(propNode.argument);
-      } else if (propNode.type === 'JSXIdentifier') {
-        return propNode.name;
+      switch (propNode.type) {
+        case 'JSXSpreadAttribute':
+          return sourceCode.getText(propNode.argument);
+        case 'JSXIdentifier':
+          return propNode.name;
+        case 'JSXMemberExpression':
+          return `${propNode.object.name}.${propNode.property.name}`;
```

Comment:
This will not handle `Foo.Bar.Baz`, do we need that?

Reply:
I think it should handle dots, yes - and if bracket notation is supported, that too (but i don’t think it is)

Reply:
Do you mean `Foo['Bar']`? Yeah, looks like it's not supported according to Babel REPL.

Reply:
To clarify; it should handle any number of nested member expressions.

Reply:
I hope recursion is good enough for practical uses 😄